CONTRIBUTING.md

```diff
@@ -59,9 +59,7 @@ Here's how to make a one-off code change. (If you're working on a larger feature
   * Before making a commit, start up a local instance of Oppia and do some manual testing in order to check that you haven't broken anything! Also, ensure that all automated tests still pass:
 
     ```
-      bash scripts/test.sh
-      bash scripts/run_js_tests.sh
-      bash scripts/run_integration_tests.sh (if necessary)
+      bash scripts/run_tests.sh
```

Comment:
Actually, for me, the e2e tests (with the better parallelization) now take roughly as long to run for me as the backend tests, so I lean towards having the default be to run all of them (especially since the e2e tests tend to catch important issues). So I lean towards leaving this as-is for now.